docker_explorer/de.py

```diff
@@ -210,5 +208,9 @@ def Main(self):
     else:
       raise ValueError('Unhandled command %s' % options.command)
 
-if __name__ == '__main__':
+def main():
+  """Main function"""
   DockerExplorer().Main()
+
+if __name__ == '__main__':
```

Comment:
I would also recommend moving de.py into a scripts subdirectory e.g. https://github.com/libyal/winreg-kb

Reply:
Keeping it as is, as I don't see any other use of the "docker_explorer" module than through the de.py file.